src/Core/Components/List/ListComponentBase.razor.cs

```diff
@@ -145,6 +161,9 @@ public abstract class ListComponentBase<TOption> : FluentComponentBase where TOp
     [Parameter]
     public virtual TOption? SelectedOption { get; set; }
 
+    [Parameter]
+    public virtual FluentOption<TOption>? SelectedFluentOption { get; set; }
```

Comment:
What is the goal of this property ?

Reply:
Left over from coding efforts. Will remove